tenzir/integration/reference/node-suricata-alert/step_01.ref

```diff
@@ -1,4 +1,4 @@
-<2011-08-12T12:52:57.716360, 1031464864740687, 83, null, null, 147.32.84.165, 1181, 78.40.125.4, 6667, "TCP", "alert", null, null, "allowed", 1, 2017318, 4, "ET CURRENT_EVENTS SUSPICIOUS IRC - PRIVMSG *.(exe|tar|tgz|zip)  download command", "Potentially Bad Traffic", 2, null, null, null, null, 27, 35, 2302, 4520, 2011-08-12T12:47:24.357711, null, null, null, null, null, "UFJJVk1TRyAjemFyYXNhNDggOiBzbXNzLmV4ZSAoMzY4KQ0K", "PRIVMSG #zarasa48 : smss.exe (368)\x0D\x0A", 0, "AB5J2xnDCAAntbcZCABFAABMGV5AAIAGLlyTIFSlTih9BASdGgvw0QvAxUWHdVAY+rCL4gAAUFJJVk1TRyAjemFyYXNhNDggOiBzbXNzLmV4ZSAoMzY4KQ0K", 1>
+<2011-08-12T12:52:57.716360, 1031464864740687, 83, null, null, 147.32.84.165, 1181, 78.40.125.4, 6667, "TCP", "alert", null, null, "allowed", 1, 2017318, 4, "ET CURRENT_EVENTS SUSPICIOUS IRC - PRIVMSG *.(exe|tar|tgz|zip)  download command", "Potentially Bad Traffic", 2, null, null, null, null, 27, 35, 2302, 4520, 2011-08-12T12:47:24.357711, null, null, null, null, null, b"PRIVMSG #zarasa48 : smss.exe (368)\x0D\x0A", "PRIVMSG #zarasa48 : smss.exe (368)\x0D\x0A", 0, b"\x00\x1EI\xDB\x19\xC3\x08\x00'\xB5\xB7\x19\x08\x00E\x00\x00L\x19^@\x00\x80\x06.\\x93 T\xA5N(}\x04\x04\x9D\x1A\x0B\xF0\xD1\x0B\xC0\xC5E\x87uP\x18\xFA\xB0\x8B\xE2\x00\x00PRIVMSG #zarasa48 : smss.exe (368)\x0D\x0A", 1>
```

Comment:
Not sure if this is what want, but we probably do not care too much. This is from `export ascii`.

Reply:
We never ported that internal format to our new parser/printer framework (and don't have any plans to), so it's not worth paying much attention to this. I think this is okay for now.